### DISCUSSION TITLE
Mailgun and "recipient-variables"

### DISCUSSION THREAD

**Discussion by jamie-mindful** [magicbell/community#78] [Questions] [answered]
I'm looking at migrating an existing product that uses Mailgun to MagicBell using the Mailgun integration.

A feature used a lot in the existing product is batch sending and attaching ["recipient-variables"](https://documentation.mailgun.com/en/latest/user_manual.html?highlight=recipient-variables#batch-sending-1).

Is it possible to pass these variables to the MagicBell API which then passes it over to Mailgun? Also, is there any plans to add support to Mailgun EU any time soon as that would be really useful.

Thank you.

**Comment by kgray-MB** [comment, marked as answer]
Hey @jamie-mindful,

Regarding 'recipient-variables', you can pass data to Mailgun using provider overrides.


From our [Mailgun integration docs](https://www.magicbell.com/docs/email/mailgun): 

> Our [create notifications endpoint](https://www.magicbell.com/docs/rest-api/reference#create-notification) allows you to customize the payload sent to Mailgun by setting [notification.overrides](https://www.magicbell.com/docs/overrides).
> 
> The notification.overrides.mailgun parameter accepts all the body parameters of [Mailgun's Messages API](https://documentation.mailgun.com/en/latest/api-sending.html#sending).
>

**Comment by unamashana** [comment]
@jamie-mindful We have Mailgun EU in our roadmap, but no ETA as of now.
